configure/CMakeLists.txt

```diff
@@ -210,5 +210,13 @@ if(NOT WIN32)
         endif()
     endif()
 endif()
+
+if (TANGO_JPEG_MMX STREQUAL "false")
+  message("MMX support disabled")
+else (TANGO_JPEG_MMX STREQUAL "false")
+  message("Building MMX support; use '-DTANGO_JPEG_MMX=false' to disable")
+  set(TANGO_JPEG_MMX true)
+endif (TANGO_JPEG_MMX STREQUAL "false")
```

Comment:
I think it would be simpler to use CMake's option (and the usual ON/OFF values) here.
Please check the documentation: https://cmake.org/cmake/help/latest/command/option.html

See some examples:
https://github.com/tango-controls/cppTango/blob/3343f94d4151ab1dbb7eee7f5adb5cf48678db4d/CMakeLists.txt#L7-L9
https://github.com/tango-controls/cppTango/blob/3343f94d4151ab1dbb7eee7f5adb5cf48678db4d/configure/CMakeLists.txt#L235

Reply:
Much better, indeed! I'll fix it Monday.

Reply:
Fixed in https://github.com/tango-controls/cppTango/pull/657/commits/acc962bcec595d1e28a0a6a7158e616e317d19d9. Tested on PPC and x86_64 (Ubuntu 18.04)